src/app/layout.tsx

```diff
@@ -1,17 +1,15 @@
 import type { Metadata } from "next";
-import { Geist, Geist_Mono } from "next/font/google";
+import localFont from "next/font/local";
 import "./globals.css";
 import {ThemeProvider} from "@/components/theme-provider";
 
-const geistSans = Geist({
-  variable: "--font-geist-sans",
-  subsets: ["latin"],
-});
+const geistSans = localFont({
+  src: '../../public/Geist-Regular.ttf',
```

Comment:
This doesn't work and results in body having two "undefined" css classes
```suggestion
  src: '../../public/Geist-Regular.ttf',
  variable: "--font-geist-sans"
```

Reply:
Although it doesn't matter too much because the fonts have no effect in the browser for some reason even on main